src/views/CompleteRegistrationComponent.tsx

```diff
@@ -278,9 +282,9 @@ export default function CompleteRegistrationComponent() {
             <Button
               variant="contained"
               sx={{ alignSelf: 'center' }}
-              onClick={() => window.location.assign(ENV.URL_FE.LANDING)}
+              onClick={() => window.location.assign(ENV.URL_FE.ASSISTANCE)}
```

Comment:
The assistance is reachable only if the user is logged, otherwise the email manager of the client should be opened prefilled the email of the support.
just use:
import { buildAssistanceURI } from '../../services/assistanceService';

@see: https://github.com/pagopa/selfcare-common-frontend/blob/main/src/lib/components/Footer/Footer.tsx#L129